src/voter/mod.rs

```diff
@@ -134,6 +134,19 @@ pub enum CommitProcessingOutcome {
 	Bad(BadCommit),
 }
 
+#[cfg(any(test, feature = "test-helpers"))]
+impl CommitProcessingOutcome {
+	/// Returns a `Good` instance of commit processing outcome's opaque type. Useful for testing.
+	pub fn good() -> CommitProcessingOutcome {
```

Comment:
can we guarantee that we won't need to break API for these?

Reply:
I think we should always be able to provide some default value for this. As we add more fields to it we might need to add some different helper methods to create specific instances though. Is this what you were thinking?

Reply:
Yeah -- mostly that I wasn't convinced that we can guarantee a default